pkg/deployments/tasks/20230130-composable-stable-pool-v3/test/test.fork.ts

```diff
@@ -0,0 +1,333 @@
+import hre from 'hardhat';
+import { expect } from 'chai';
+import { BigNumber, Contract } from 'ethers';
+
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { describeForkTest } from '../../../src/forkTests';
+import Task, { TaskMode } from '../../../src/task';
+import { getForkedNetwork } from '../../../src/test';
+import { getSigner, impersonate } from '../../../src/signers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+
+import { MAX_UINT256, ZERO_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+import { bn, fp } from '@balancer-labs/v2-helpers/src/numbers';
+import { BasePoolEncoder, StablePoolEncoder, SwapKind } from '@balancer-labs/balancer-js';
+import { actionId } from '@balancer-labs/v2-helpers/src/models/misc/actions';
+import { expectEqualWithError } from '@balancer-labs/v2-helpers/src/test/relativeError';
+
+describeForkTest.only('ComposableStablePool', 'mainnet', 16492500, function () {
```

Comment:
```suggestion
describeForkTest('ComposableStablePool', 'mainnet', 16492500, function () {
```

Remove `.only` when ready.